### PR TITLE
Added cutting, slicing, dicing.

### PR DESCRIPTION
**Pull request by mpomarlan**
Solves issue #94

### CODE REVIEW

File: owl/SOMA-ACT.owl [hunk 1]
```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns="http://www.ease-crc.org/ont/SOMA.owl#"
-     xml:base="http://www.ease-crc.org/ont/SOMA.owl"
-     xmlns:SOMA="http://www.ease-crc.org/ont/SOMA.owl#"
-     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+<rdf:RDF xmlns="http://www.ease-crc.org/ont/SOMA-ACT.owl#"
+     xml:base="http://www.ease-crc.org/ont/SOMA-ACT.owl"
```
Comment (daniel86) [inline owl/SOMA-ACT.owl:3]: Protege has changed back the xml:base :(
How can we avoid that?
It must be that it overwrites it with the ontology rdf:about?

Comment (daniel86) [reply owl/SOMA-ACT.owl:3]: I guess most pragmatic and easy would be to change all ontology rdf:about to SOMA.owl

Comment (daniel86) [reply owl/SOMA-ACT.owl:3]: this would change all the rdf:about's: https://github.com/ease-crc/soma/pull/97

Comment (daniel86) [reply owl/SOMA-ACT.owl:3]: does not seem like an option because Protege would not load two modules next to each other anymore :(

Comment (daniel86) [reply owl/SOMA-ACT.owl:3]: ok, change of plan. part of the refactoring must happen in the uglify step

File: owl/SOMA-ACT.owl [hunk 2]
```diff
@@ -179,6 +179,33 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-ACT.owl#Cutting -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ACT.owl#Cutting">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#ModifyingPhysicalObject"/>
+        <rdfs:comment>The goal of this task is to separate some target object into several pieces by means of cutting into its constituent material. Unlike a disassembly, a cut is usually not easily reversible.</rdfs:comment>
```
Comment (daniel86) [inline owl/SOMA-ACT.owl:186]: several -> one or more

File: owl/SOMA-ACT.owl [hunk 3]
```diff
@@ -179,6 +179,33 @@
...
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA-ACT.owl#Dicing -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ACT.owl#Dicing">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA-ACT.owl#Cutting"/>
+        <rdfs:comment>A particular kind of cutting where the goal is to produce small pieces out of some object or material. Unlike slices, the pieces to be obtained do not have one or two dimensions being more prominent than others. &quot;Dice&quot;, the pieces dicing results in, are approximately cubic.</rdfs:comment>
```
Comment (daniel86) [inline owl/SOMA-ACT.owl:195]: at a later point we could axiomatize the cubic shape

**Comment by mpomarlan** [comment]
I changed the concept IRIs for Cutting, Slicing, Dicing to be SOMA.owl rather than SOMA-ACT.owl

As you noticed, changing the xml:base etc tags wrecks Protege's ability to actually import dependencies, so for now I kept the xml:base for SOMA-{ACT,ELAN} to be also SOMA-{ACT,ELAN}.

I suggest we revert the xml:base and xmlns in every other file as well.

An observation -- though not the most ergonomic thing ever, it is possible to customize Protege's entity creation preferences and specify a particular IRI for new entities. It's a possible workaround.

**Comment by daniel86** [comment]
> I changed the concept IRIs for Cutting, Slicing, Dicing to be SOMA.owl rather than SOMA-ACT.owl
> 
> As you noticed, changing the xml:base etc tags wrecks Protege's ability to actually import dependencies, so for now I kept the xml:base for SOMA-{ACT,ELAN} to be also SOMA-{ACT,ELAN}.
> 
> I suggest we revert the xml:base and xmlns in every other file as well.
> 
> An observation -- though not the most ergonomic thing ever, it is possible to customize Protege's entity creation preferences and specify a particular IRI for new entities. It's a possible workaround.

yes I realized that as well. sorry for the confusion.
It does not matter anymore as long as the IRI starts with "http://www.ease-crc.org/ont/SOMA".
This is because the uglify step does change all such IRIs to the one SOMA IRI automatically.
So, yes, we can set the xmlns back in all files.

**Pull request merged**